app/views/shared/_email_languages.html.erb

```diff
@@ -3,26 +3,9 @@ locals:
 * f: from validated_form_for
 * selection: the current language selection
 %>
-<div class="margin-bottom-4">
+<div class="margin-bottom-2 col-8">
   <ul class="usa-input-list">
-    <% I18n.available_locales.each do |locale| %>
-      <% item_id = "email-locale-#{locale}" %>
-      <li class="grid-col-8 mobile-lg:grid-col-6">
-        <%= f.input_field(:email_language,
-                          type: :radio,
-                          value: locale,
-                          checked: selection ?
-                            selection.to_s == locale.to_s :
-                            (I18n.locale.to_s == locale.to_s),
-                          class: 'usa-radio__input',
-                          id: item_id) %>
-        <%= f.label(:email_language,
-                    locale == I18n.locale ?
-                      t("account.email_language.default", language: t("i18n.locale.#{locale}")) :
-                      t("i18n.locale.#{locale}"),
-                    for: item_id,
-                    class: 'usa-radio__label usa-radio-bordered') %>
-      </li>
-    <% end %>
+    <%= f.select :email_language, I18n.available_locales.
+        map {|locale| [t("i18n.locale.#{locale}"), locale.to_s]}, {},  class: 'usa-select col-6' %>
```

Comment:
looks like `selection:` is now unused? does it correctly guess from `user.email_language`? If so, can we remove option from this partial?

Reply:
UI folks changed mind on dropdown...reverting